portfolio/src/main/webapp/index.html

```diff
@@ -1,19 +1,182 @@
+                <img class="img-container" style="background-image:url(images/main.jpg)"/>
+            </div> 
+        </div>
+        <div> 
+            <ul class="text hidden">
+                <li>C</li>
+                <li class="ghost">A</li>
+                <li class="ghost">R</li>
+                <li class="ghost">L</li>
+                <li class="ghost">O</li>
+                <li class="spaced">A</li>
+                <li class="ghost">N</li>
+                <li class="ghost">G</li>
+                <li class="ghost">E</li>
+                <li class="ghost">L</li>
+                <li class="spaced">P</li>
+                <li class="ghost">o</li>
+                <li class="ghost">r</li>
+                <li class="ghost">t</li>
+                <li class="ghost">f</li>
+                <li class="ghost">o</li>
+                <li class="ghost">l</li>
+                <li class="ghost">i</li>
+                <li class="ghost">o</li>
+            </ul>
+        </div>
+        <div class="content-wrapper">
+            <div class="portfolio-items-wrapper">
+                <div class="portfolio-item-wrapper">
+                    <div class="portfolio-img-background" style="background-image:url(images/projects.jpg)"></div>
+                    <div class="img-text-wrapper">
+                        <div class="logo-wrapper">
+                            <a href="projects.html">
+                                <img src="images/projects.png" alt="">
+                            </a>
+                        </div>
+                        <div class="subtitle">Learn about my projects
+                        </div>
+                    </div>
+                </div>
+
+                <div class="portfolio-item-wrapper">
+                    <div class="portfolio-img-background" style="background-image:url(images/aboutme.jpg)"></div>
+                    <div class="img-text-wrapper">
+                        <div class="logo-wrapper">
+                            <a href="about.html">
+                                <img src="images/perfil.png" alt="">
+                            </a>
+                        </div>
+                        <div class="subtitle">Learn about me
+                        </div>
+                    </div>
+                </div>
+
+                <div class="portfolio-item-wrapper">
+                    <div class="portfolio-img-background" style="background-image:url(images/coding1.jpg)"></div>
+                    <div class="img-text-wrapper">
+                        <div class="logo-wrapper">
+                            <a href="https://www.linkedin.com/in/carlo-angel-lujan-garcia/" target="_blank" >
+                                <img src="images/socialmedia.png" alt="">
+                            </a>
+                        </div>
+                        <div class="subtitle">Instagram: @carlo_lj <br>Github:CarloLj <br>Facebook:Carlo Angel Lj</div>
```

Comment:
Instead of using <br> look into using another DOM element, <p> or nested <div> to manage the new lines. Using br isn't [semantic](https://en.wikipedia.org/wiki/Semantic_Web). If you want things to be separated best practice is to make them separate.

```suggestion
                        <div class="subtitle">
                             <p>Instagram: @carlo_lj</p>
                             <p>Github:CarloLj</p>
                             <p>Facebook:Carlo Angel Lj</p>
                        </div>
```

Reply:
Okay! Working on it!